example/src/scenarios/get-pid.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Credential,
+  createCryptoContextFor,
+  type IdentificationContext,
+  type IntegrityContext,
+} from "@pagopa/io-react-native-wallet";
+import { error, result, toResultOrReject } from "./types";
+import getAttestation from "./get-attestation";
+import { openAuthenticationSession } from "@pagopa/io-react-native-login-utils";
+import {
+  REDIRECT_URI,
+  WALLET_PID_PROVIDER_BASE_URL,
+  WALLET_PID_PROVIDER_REDIRECT_OVERRIDE_URL,
+} from "@env";
+import uuid from "react-native-uuid";
+import { generate } from "@pagopa/io-react-native-crypto";
+
+export default (integrityContext?: IntegrityContext) => async () => {
+  try {
+    if (!integrityContext) {
+      return error("Integrity context not available");
+    }
```

Comment:
Why is it optional if it is then required?

Reply:
Fixed in 4ca4c99d1ebdd84e043deda445b3b13a12005724, thanks.